core/src/main/kotlin/com/malinskiy/marathon/report/stdout/StdoutReporter.kt

```diff
@@ -10,21 +10,49 @@ class StdoutReporter(private val timer: Timer) : Reporter {
         val summary = executionReport.summary
         if (summary.pools.isEmpty()) return
 
-        val cliReportBuilder = StringBuilder().appendln("Marathon run finished:")
-        summary.pools.forEach {
-            cliReportBuilder.appendln(
-                "Device pool ${it.poolId.name}:\n" +
-                        "\t${it.passed} passed, ${it.failed} failed, ${it.ignored} ignored tests\n" +
-                        "\tFlakiness overhead: ${it.rawDurationMillis - it.durationMillis}ms\n" +
-                        "\tRaw: ${it.rawPassed} passed, ${it.rawFailed} failed, ${it.rawIgnored} ignored, ${it.rawIncomplete} incomplete tests"
-            )
+        val cliReportBuilder = StringBuilder().appendLine("Marathon run finished:")
+        summary.pools.forEach { poolSummary ->
+            cliReportBuilder.appendLine("Device pool ${poolSummary.poolId.name}:")
+            cliReportBuilder.appendLine("\t${poolSummary.passed.size} passed, ${poolSummary.failed.size} failed, ${poolSummary.ignored.size} ignored tests")
+
+            if(poolSummary.failed.isNotEmpty()){
+                cliReportBuilder.appendLine("\tFailed tests:")
+                poolSummary.failed
+                    .toSortedSet()
+                    .forEach { testName -> cliReportBuilder.appendLine("\t\t$testName") }
+            }
+
+            cliReportBuilder.appendLine("\tFlakiness overhead: ${poolSummary.rawDurationMillis - poolSummary.durationMillis}ms")
+            cliReportBuilder.appendLine("\tRaw: ${poolSummary.rawPassed.size} passed, ${poolSummary.rawFailed.size} failed, ${poolSummary.rawIgnored.size} ignored, ${poolSummary.rawIncomplete.size} incomplete tests")
+
+            if(poolSummary.rawFailed.isNotEmpty()){
+                cliReportBuilder.appendLine("\tFailed tests:")
+                poolSummary.rawFailed
+                    .groupBy { it }
+                    .toSortedMap()
+                    .mapValues { it.value.size }
+                    .forEach { (testName, count) ->
+                        cliReportBuilder.appendLine("\t\t$testName failed $count times")
```

Comment:
Since we don't have proper plular support, maybe reword to `time(s)`?

Reply:
WDYT about linguistic library implementation such as https://github.com/atteo/evo-inflector?

Reply:
Can we just use `(s)`? we're not a GUI app, please just change the template to use `(s)`

Reply:
No problem mate